tests/Application/config/routes/sylius_admin_api.yaml

```diff
@@ -1,3 +0,0 @@
-sylius_admin_api:
```

Comment:
This file has no symbolic link ?

Reply:
this file was [removed in 1.10](https://github.com/Sylius/Sylius/commit/5f3d5a8273cc4639b40037a135259c4b5ac1bada#diff-01734a8d027dc63fd22c87d8937ad428f3ab3798a489d32dc0a22fbed63395b8)